Update GET top N api documentation (#8139)
opensearch-project/documentation-website: _observing-your-data/query-insights/top-n-queries.md

Reviewer: Monitoring the top N queries in query insights features can help you gain real-time insights into the top queries with high latency within a certain time frame (for example, the last hour).

need to included cpu and memory too here

Reviewer 2: +1, let's update this in this PR.

# Top N queries

> Monitoring the top N queries using query insights allows you to gain real-time visibility into the queries with the greatest latency or resource consumption over a specified time period (for example, the last hour).

Reviewer 3: ```suggestion
Monitoring the top N queries using query insights allows you to gain real-time visibility into the queries with the highest latency or resource consumption in a specified time period (for example, the last hour).
```

```
{% include copy-curl.html %}

> If you are not seeing results, verify top N is enabled for the target metric type and ensure search requests were sent during the current time window.

Reviewer 4: @dzane17 Here's a rewording suggestion. Could you also clarify what "current" means in this context?
```suggestion
If your query returns no results, ensure that top N is enabled for the target metric type and that search requests were made within the current time window.
```

> If your query returns no results, ensure that top N is enabled for the target metric type and that search requests were made within the current [time window](#configuring-the-window-size).

Reviewer: Do we want to mention that the sorting of the results will be based on the metric type provided?

```
{% include copy-curl.html %}

> If your query returns no results, ensure that top N is enabled for the target metric type and that search requests were made within the current [time window](#configuring-the-window-size).

Reviewer 3: Should "top N" be in code font or otherwise specially formatted?

Reviewer 4: No. But I clarified so it reads better.

> If your query returns no results, ensure that top N is enabled for the target metric type and that search requests were made within the current [time window](#configuring-the-window-size).

Reviewer 4: ```suggestion
If your query returns no results, ensure that top N query monitoring is enabled for the target metric type and that search requests were made within the current [time window](#configuring-the-window-size).
```
